pegasus/sites.v3/code.org/public/js/volunteer-local.js

```diff
@@ -209,6 +227,87 @@ function loadMap(locations) {
   }
 
   gmap.Load(mapOptions);
+
+  if (window.location.search.indexOf("mapbox") !== -1) {
+    $("#mapbox-container").show();
+
+    mapboxMap = new mapboxgl.Map({
+      container: "mapbox",
+      style: "mapbox://styles/codeorg/cjyudafoo004w1cnpaeq8a0lz",
+      center: [lng, lat],
+      zoom: 12
+    });
+
+    let features = [];
+    if (mapOptions.locations && mapOptions.locations.length > 0) {
+      for (var location of mapOptions.locations) {
+        const feature = {
+          type: "Feature",
+          properties: {
+            description: location.html,
+            "icon-image": "star-marker",
+            "icon-size": 22
+          },
+          geometry: {
+            type: "Point",
+            coordinates: [location.lon, location.lat]
+          }
+        };
+        features.push(feature);
+      }
+    }
+
+    mapboxMap.on("load", function() {
+      mapboxMap.loadImage(
+        "https://maps.google.com/mapfiles/kml/paddle/red-stars.png",
+        (error, image) => {
+          if (error) {
+            console.log(error);
```

Comment:
Is it worth reporting this to NewRelic or something? (Could be in a followup PR)

Reply:
Good idea.  I might punt to a future PR when we get this ready to ship, and we can work out a way to do it consistently across all our implementations.